projects/angular-material-extensions/select-country/src/lib/i18n/de.ts

```diff
@@ -811,7 +811,7 @@ export const COUNTRIES_DB_DE: Country[] =
     },
```

Comment:
this is true

Reply:
Yeah, was seeing that error when trying to get Norway country and noticed the wrong alpha2Code.